server/lib/nl/fulfillment/types.py

```diff
@@ -84,8 +84,11 @@ class SV2Thing:
 class ExistInfo:
   is_single_point: bool = False
   # Facet metadata where keys are metadata keys and values are metadata values.
-  # Keys include 'facetId' and optional 'unit'.
+  # Keys include 'facetId'. 'earliestDate', 'latestDate', and optional 'unit'
+  # and 'observationPeriod'.
   facet: Dict[str, str] = field(default_factory=dict)
+  # Latest valid date that there exists data for.
+  latest_valid_date: str = ''
```

Comment:
nit: Used only if single-date/date-range is set?  document?

Reply:
done